astrology.lic

```diff
@@ -279,6 +279,9 @@ class Astrology
 
     if @have_telescope
       case DRCMM.center_telescope(body)
+      when /Center what?/
```

Comment:
❓ Does this work without a 15 second delay? I don't see `Center what?` in the list of matching messages in `DRCMM.center_telescope`

Reply:
Should be down below.